src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -2555,23 +2556,16 @@ public async Task SendAsync_RequestVersionNotSpecified_ServerReceivesVersion11Re
             Assert.Equal(new Version(1, 1), receivedRequestVersion);
         }
 
-        [ActiveIssue(23037, TestPlatforms.AnyUnix)]
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Specifying Version(2,0) throws exception on netfx")]
         [OuterLoop("Uses external server")]
-        [Theory]
+        [ConditionalTheory]
```

Comment:
I don't think `ConditionalTheory` is needed - as no parameter is passed in. `[Theory]` should work.

Reply:
That is needed for dynamic Skip via SkipTestException